docs/api/browser-window.md

```diff
@@ -1102,8 +1102,14 @@ On Linux always returns `true`.
 * `flag` Boolean
 * `level` String (optional) _macOS_ - Values include `normal`, `floating`,
   `torn-off-menu`, `modal-panel`, `main-menu`, `status`, `pop-up-menu`,
-  `screen-saver`, and ~~`dock`~~ (Deprecated). The default is `floating`. See the
-  [macOS docs][window-levels] for more details.
+  `screen-saver`, and ~~`dock`~~ (Deprecated). The default is `floating` when
+  `flag` is true. The `level` is reset to `normal` when the flag is false. See
+  the [macOS docs][window-levels] for more details.
+* `level` String (optional) _Windows_ - Values are the same as on macOS to make
```

Comment:
This shouldn't be the a different parameter with the same name; that'll confuse our documentation parser. This should just specify what values of the existing `level` parameter now apply to Windows.

Reply:
Done.